components/atoms/HeaderLang.js

```diff
@@ -30,7 +30,7 @@ export function HeaderLang() {
   //
 
   return (
-    <section id="wb-lng" className="text-right">
+    <section id="wb-lng" className="col-xs-3 col-sm-12 pull-right text-right">
```

Comment:
the language switcher has a "question mark" cursor right now -- it should always have a pointer finger.

Reply:
yes, same as canada.ca . I'll change it anyways

Reply:
You're right, Canada.ca does do it. I think the cursor makes more sense though.